### PR TITLE
Add support to accept Map as part of mapBlockOf method

### PR DESCRIPTION
**Pull request by harsharastogi**
Test plan - (Please fill in how you tested your changes)

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-tests/src/main/java/com/facebook/presto/tests/StructuralTestUtil.java [hunk 1]
```diff
@@ -83,6 +84,19 @@ public static Block arrayBlockOf(Type elementType, Object... values)
         return blockBuilder.build();
     }
 
+    public static Block mapBlockOf(Type keyType, Type valueType, Map<?, ?> value)
```
Comment (jainxrohit) [inline presto-tests/src/main/java/com/facebook/presto/tests/StructuralTestUtil.java:87]: Is this being used anywhere?

Comment (harsharastogi) [reply presto-tests/src/main/java/com/facebook/presto/tests/StructuralTestUtil.java:87]: It is used in a  presto-facebook PR

Comment (harsharastogi) [reply presto-tests/src/main/java/com/facebook/presto/tests/StructuralTestUtil.java:87]: https://github.com/facebookexternal/presto-facebook/pull/1949

Comment (arunthirupathi) [reply presto-tests/src/main/java/com/facebook/presto/tests/StructuralTestUtil.java:87]: The Map's iteration order is undefined and it varies from one map implementation to the other.  Using this construct might result in more confusion.

Can the test use ? public static Block mapBlockOf(Type keyType, Type valueType, Object[] keys, Object[] values) which has more predictable ordering.

Comment (harsharastogi) [reply presto-tests/src/main/java/com/facebook/presto/tests/StructuralTestUtil.java:87]: The test cases don't care about order of entries inside the output mapBlock after merging. We convert the output mapBlock in the test case to a Map and compare it to the expected Map

Comment (arunthirupathi) [reply presto-tests/src/main/java/com/facebook/presto/tests/StructuralTestUtil.java:87]: Should it just use the Object[] overloads then ?

**Pull request closed without merging**